com.microsoft.mrtk.input/Interactors/InteractorVisuals/SpatialMouse/MouseCursor.png.meta

```diff
@@ -0,0 +1,132 @@
+fileFormatVersion: 2
```

Comment:
Out of curiosity, is this the same cursor from MRTK2? https://github.com/microsoft/MixedRealityToolkit-Unity/blob/main/Assets/MRTK/StandardAssets/Textures/MouseCursor.png

Reply:
It looks like the same asset.

Reply:
Great!